python/packages/ai/teams/app.py

```diff
@@ -806,9 +806,14 @@ async def _start_long_running_call(
             and ActivityTypes.message == context.activity.type
             and self._options.long_running_messages
         ):
+            def wrapped_callback(ctx: TurnContext) -> Awaitable:
```

Comment:
if the new activity is not being created using the old activity that would be a bug in the botframework

Reply:
I copied this logic from the [JS library code](https://github.com/microsoft/teams-ai/blob/main/js/packages/teams-ai/src/Application.ts#L1060-L1063), which made me think it's a bug in this library.

Reply:
im following up with botframework on this, will get back to you with an answer.

Reply:
I followed up with BF and they said there is no need to copy the activity over

Reply:
Do you mean they're fixing something on their end?